.github/workflows/ci.yaml

```diff
@@ -1,11 +1,12 @@
 # Check to_sandbox.txt and to_production.txt do not contain typos
-name: Google Fonts Test Server Files
+name: Continuous Test + Deploy
```

Comment:
Since this CI doesn't deploy, is there a better name for this?

Reply:
We could just remove "+ Deploy"?

Reply:
I'll send a PR